tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py26, py27, py33, py34, py35, pypy, pypy3
+envlist = py26, py27, py33, py34, py35,i py36, pypy, pypy3
```

Comment:
What does the `i` do here? Thought that would make Travis fail, but it has not. :blush:

Reply:
oops, will fix